src/gmt_map.c

```diff
@@ -9435,6 +9443,8 @@ int gmt_map_setup (struct GMT_CTRL *GMT, double wesn[]) {
 
 	if ((error = gmt_proj_setup (GMT, wesn)) != GMT_NOERROR) goto gmt_map_setup_end;
 
+	gmt_set_undefined_defaults (GMT, MAX (GMT->current.map.width, GMT->current.map.height));	/* We must change any undefined defaults given max plot dimension */
```

Comment:
Would it be possible to skip ```gmt_set_undefined_defaults``` if in classic mode and nothing is set to "auto"? Otherwise commands like ```gmt psbasemap -R-90/-70/0/20 -JM5c -Baf > test.ps``` 
give a notice that 
```psbasemap [NOTICE]: Computed primary annotation font size: 8.34293 p  Dimension scaling: 0.834293```, which might lead to concerns when users run their old scripts that the default behavior is changing and gmt.conf defaults are being overwritten.

Reply:
Possibly, but remember that some of these messages you look at were deliberately giving status of NOTICE so I could more easier see them without doing -Vd.  These should eventually be changed to GMT_MSG_DEBUG and then it probably does not matter?

Reply:
Ok